cmake/Modules/Findlibdbi.cmake

```diff
@@ -0,0 +1,13 @@
+# Determine compiler flags for libdbi
+# Once done this will define
+# LIBDBI_FOUND - libdbi found
+
+find_path(LIBDBI_INCLUDE_DIR NAMES dbi.h HINTS "${LIBDBI_DIR}/include/dbi")
+
+find_library(LIBPGSQL_DRIVER_LIBRARY NAMES dbdpgsql HINTS "${LIBDBI_DIR}/lib/${CMAKE_C_LIBRARY_ARCHITECTURE}/dbd")
+
+find_library(LIBDBI_DEV_LIBRARY NAMES dbi)
```

Comment:
Karthik - these are libraries installed thro' package manager. Unless we build libraries from source, we shouldn't have to worry. What do you think?